### DISCUSSION TITLE
Instruments with images (wikidata)

### DISCUSSION THREAD

**Discussion by kyrieb-ekat** [DDMAL/VIM#77] [General]
october 604 item (HBS with image)
https://mcgill-my.sharepoint.com/:x:/g/personal/kyrie_bouressa_mail_mcgill_ca/EUOaffpqF4dIgUfXdjHaCbwBtz3zQDKhGLRVY6uHj80WTg?e=P6a4hf

**Comment by kyrieb-ekat** [comment]
Here's a .csv with some initial number to term mappings. I have tried to keep these terms relatively broad. Please let me know if there are any issues with the attachment, or have any questions about the numbers. 
@yinanazhou @dchiller @Monalisa98927 
[VIM - HBS Term Mapping .csv](https://github.com/DDMAL/VIM/files/13299767/VIM.-.HBS.Term.Mapping.csv)

**Reply by dchiller** [reply]
Cool! Also added this file to the comments in #85.
